pkg/gui/controllers/workspace_reset_controller.go

```diff
@@ -64,6 +64,28 @@ func (self *FilesController) createResetMenu() error {
 			},
 			Key: 'c',
 		},
+		{
+			LabelColumns: []string{
+				self.c.Tr.LcDiscardStagedChanges,
+				red.Sprint("stash staged and drop stash"),
+			},
+			Tooltip: "This will create a new stash entry containing only staged files and then drop it.",
```

Comment:
```suggestion
			Tooltip: "This will create a new stash entry containing only staged files and then drop it, so that the working tree is left with only unstaged changes",
```
We'll want to i18n-ise this